eatin-client/src/homePage/HomePage.tsx

```diff
@@ -81,6 +89,7 @@ export const HomePage: FC = () => {
 
     return (
         <div>
+            <AsyncDataLoaderWrapper loading={loading} text="loading recipes...">
```

Comment:
instead of the state `loading`, put here the value of this state which is `!chickenLoading && !cakesLoading && !japaneseLoading && !greekLoading`